test/performance/scripts/create-outgoing-payments.js

```diff
@@ -0,0 +1,182 @@
+import http from 'k6/http'
+export const options = {
+  // A number specifying the number of VUs to run concurrently.
+  vus: 10,
+  // A string specifying the total duration of the test run.
+  duration: '30s'
```

Comment:
Should we bump this up? Im seeing this:

> time="2024-08-05T18:51:25Z" level=warning msg="No script iterations fully finished, consider making the test duration longer"

Reply:
Looks like I need to bump it up to match the [issue description](https://github.com/interledger/rafiki/issues/2810):

> For now, we can set the scenario to be quite small, a single virtual user (VU) across 10 minutes. We should be able to see the local playground Grafana dashboard properly populate with metrics (and traces).